build/media-suite_compile.sh

```diff
@@ -707,6 +708,16 @@ if [[ $standalone = y && $faac = y ]] && ! files_exist "${_check[@]}" &&
     do_checkIfExist
 fi
 
+_check=(bin-audio/exhale.exe)
+if [[ $standalone = y && $exhale = y ]] &&
+    do_vcs "https://gitlab.com/ecodis/exhale"; then
+	do_patch "https://gist.githubusercontent.com/moisespr123/abac078f080cca10d605ea0390323bc1/raw/exhale-patch"
```

Comment:
You might want to try to upstream this if possible, it would be easier to get upstream support for mingw-w64 if their stuff actually compiles after all.

PS: Noticed your issue https://gitlab.com/ecodis/exhale/-/issues/5

Based on their README, they don't even use anything other than personal email for contributions

```none
If you are interested in contributing to exhale, please email one of
the developers. Merge requests with fixes and/or speedups are highly
appreciated.
```

Reply:
```suggestion
    do_patch "https://gist.githubusercontent.com/moisespr123/abac078f080cca10d605ea0390323bc1/raw/exhale-patch"
```
also, this is using a tab